components/tidb_query_vec_executors/src/runner.rs

```diff
@@ -53,6 +54,13 @@ pub struct BatchExecutorsRunner<SS> {
 
     exec_stats: ExecuteStats,
 
+    /// Minimum rows to return in batch stream mode.
+    streaming_batch_limit: usize,
```

Comment:
This field name confuses me. This field should be a upper bound of each response's row count.